pages.fr/common/apm.md

```diff
@@ -0,0 +1,17 @@
+# apm
+
+> Manageur de paquets pour Atom.
```

Comment:
```suggestion
> Gestionnaire de paquets pour Atom.
```

Reply:
That true, `Gestionnaire` is better than `Manageur`